cr.sh

```diff
@@ -365,6 +377,10 @@ release_chart() {
 }
 
 helm_login() {
```

Comment:
I see it's not there, but I guess it's a better option to check for absence both of username and password, without the new variable.

Reply:
So if you don't please change. If not just let me know and I'll approve and merge. Thank you @bramk

Reply:
Hi @dennybaa, sure that would be fine for me too. I'll update it.

Reply:
Sorry @dennybaa I abandoned this effort. Merge or close? Your call :)